src/View/Admin/course_inscription.html.twig

```diff
@@ -4,5 +4,9 @@
         {% include 'Admin/Includes/_navbar.html.twig' %}
     </header>
 {% endblock %}
-{% block content %}
+{% block content %} 
+    <main>
+        <h1> Administration: Formulaires d'inscription aux cours </h1>
```

Comment:
spaces around text? + is it really forms that we're gonna find here? or inscription sheets?

Reply:
Done